README.md

```diff
@@ -1,45 +1,30 @@
-# pangraph
-
-[![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
-
-This is a Haskell Library for parsing graph formats such as [GraphML](http://graphml.graphdrawing.org/) and the XML files produced by [Workcraft](https://www.workcraft.org/).
-The library currently implements parsing only the nodes and edges of undirected graphs.  
-See the User Guide for how to use this, with examples below.
-Information on current graph support can be found below that.
-
-## How to use Pangraph
-
-Pangraph offers an api in the module `Pangraph` for accessing of graphs including constructors and getters.  
-You can construct entire graph or edit the results of parser.
-These are imported independently form their own modules:  
+# pangraph [![Build Status](https://travis-ci.org/tuura/pangraph.svg?branch=master)](https://travis-ci.org/tuura/pangraph)  
+
+Pangraph is a Haskell library which offers for parsing and serializations for graph files. As well as conversions to other Haskell graph formats. An example format is [GraphML](http://graphml.graphdrawing.org/). A graphml file
+could for example represent the following graph, with vertices from `A` to `E` and connections, the edges, between them.
+![a-sample-graph](examples/graphs/graph-AE.png?raw=true)
+Source:
```

Comment:
No need to acknowledge the source of the image in this case since I'm a co-author in both cases.